apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/ListObjectsTest.cs

```diff
@@ -72,6 +72,17 @@ public async Task PrefixAndDelimiter(string prefix, string expectedNames)
             await AssertObjects(prefix, options, expectedNames.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries));
         }
 
+        [Theory]
+        [InlineData("la*", "large.txt")]
+        [InlineData("a/*.txt", "a/o1.txt,a/o2.txt")]
+        [InlineData("a/x/*.txt", "a/x/o3.txt,a/x/o4.txt")]
+        public void MatchGlob(string globPattern, string expectedNames)
```

Comment:
I believe if you make this 

```csharp
public void MatchGlob(string globPattern, params string[] expectedNames)
```

you can just list the expected names directly, without needing to split on commas in `AssertObjectNames`.